### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by hestonhoffman**
Updates the PR to mention the merge queue.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -6,9 +6,13 @@
 ### Merge instructions
 <!-- If you want us to merge this PR as soon as we've reviewed, check the box below. If you're waiting for a release or there are other considerations that you want us to be aware of, list them below. -->
 
-- [ ] Please merge after reviewing
+Merge queue is enabled in this repo. If you'd like your PR to be merged immediately after it has receieved the required reviews, create the PR and then post the following comment:
```
Comment (brett0000FF) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
Merge queue is enabled in this repo. If you'd like your PR to be merged immediately after it has received the required reviews, create the PR and then add the following comment:
```

Comment (brett0000FF) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: I don't have a strong opinion on "post" -> "add" but it sounds a bit more natural to me.

**Review by brett0000FF — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -6,9 +6,13 @@
 ### Merge instructions
 <!-- If you want us to merge this PR as soon as we've reviewed, check the box below. If you're waiting for a release or there are other considerations that you want us to be aware of, list them below. -->
 
-- [ ] Please merge after reviewing
+Merge queue is enabled in this repo. If you'd like your PR to be merged immediately after it has received the required reviews, create the PR and then add the following comment:
```
Comment (kayayarai) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:
```

**Review by kayayarai — APPROVED** [review]

**Pull request merged**